src/SFML/Window/CMakeLists.txt

```diff
@@ -116,6 +116,12 @@ elseif(SFML_OS_LINUX OR SFML_OS_FREEBSD)
             ${SRCROOT}/FreeBSD/JoystickImpl.cpp
             ${SRCROOT}/FreeBSD/JoystickImpl.hpp
         )
+    elseif(SFML_OS_OPENBSD)
+        set(PLATFORM_SRC
+            ${PLATFORM_SRC}
+            ${SRCROOT}/Android/JoystickImpl.cpp
```

Comment:
Since the android implementation is a dummy one, I'd rather see those files being copied to a new directory.

Reply:
Ok will do later in the day fair enough :-)